### DISCUSSION TITLE
Bootshop pro

### DISCUSSION THREAD

**Discussion by afagard** [kondasoft/ks-bootshop#86] [Q&A]
https://www.kondasoft.com/products/ks-bootshop-pro goes to a 404

Is the pro version available?

**Comment by alvinkonda** [comment]
Hello,
Please check the demo from the link below. At the moment, we are deciding to sell it through a different marketplace, but if you are interested in it please let me know and I will be happy to assist you.

https://ks-bootshop.myshopify.com/?preview_theme_id=132279107746
